### PR TITLE
fix(files): selected files actions position on scroll/with readme block

### LINKED ISSUE #43149: Files actions multi-select header scrolls with content
https://github.com/nextcloud/server/assets/14975046/c7af3a2f-cd70-45e6-b000-f2101d0f754a

Most likely from https://github.com/nextcloud/server/pull/41972

@ShGKme

### PR DESCRIPTION
**Pull request by ShGKme**
* Resolves: #43149 

## Summary

- [x] Move thead overlay with files actions below the before (README) block
- [x] position `absolute` -> `sticky`
- [x] Adjust position
      - Cannot say what correcting position via negative margin is a beautiful solution, but I have no more ideas...

## Screenshots

🏚️ Before | 🏡 After
---|---
before-actions | after-actions 

<details>

### 🏚️ Before

https://github.com/nextcloud/server/assets/25978914/32c685f7-d25d-4aaf-ba8e-32f1ad8e34aa

### 🏡 After

https://github.com/nextcloud/server/assets/25978914/d92a1e35-755e-4bd5-b270-9a7b39798799

</details>

Also works on small screen (a11y size)

after-actions-small

## Checklist

- Code is [properly formatted](https://docs.nextcloud.com/server/latest/developer_manual/digging_deeper/continuous_integration.html#linting)
- [Sign-off message](https://github.com/src-d/guide/blob/master/developer-community/fix-DCO.md) is added to all commits
- [x] Tests ([unit](https://docs.nextcloud.com/server/latest/developer_manual/app_development/tutorial.html#unit-tests), [integration](https://docs.nextcloud.com/server/latest/developer_manual/app_development/tutorial.html#integration-tests), api and/or acceptance) are included
- [x] Screenshots before/after for front-end changes
- [x] Documentation ([manuals](https://github.com/nextcloud/documentation/) or wiki) has been updated or is not required
- [x] [Backports requested](https://github.com/nextcloud/backportbot/#usage) where applicable (ex: critical bugfixes)

### CODE REVIEW

**Review by skjnldsv — APPROVED** [review]
Code make sense! Thanks!!

**Review by susnux — APPROVED** [review]

File: apps/files/src/components/FilesListVirtual.vue [hunk 1]
```diff
@@ -339,14 +339,21 @@ export default defineComponent({
 
 		.files-list__table {
 			display: block;
+
+			&.files-list__table--with-thead-overlay {
+				// Hide the table header below the overlay
+				margin-top: calc(-1 * var(--row-height));
```
[the remark this answers is not in the document]
Comment (ShGKme) [reply apps/files/src/components/FilesListVirtual.vue:345]: This is the place I dislike the most in mi solution 🙈

**Comment by ShGKme** [comment]
Rebased onto master, recompiled assets

**Pull request merged**